src/ngraph/frontend/onnx_import/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# ******************************************************************************
+# Copyright (c) 2017-2018 Intel Copyright
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by application law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+set(ONNX_SOURCE_URL "https://raw.githubusercontent.com/onnx/onnx/v1.1.2/onnx/onnx.proto")
+
+set_source_files_properties(onnx.pb.h onnx.pb.cc PROPERTIES GENERATED TRUE)
+
+add_custom_command(OUTPUT onnx.proto
+    COMMAND wget --timeout=60 ${ONNX_SOURCE_URL})
```

Comment:
wget is not portable. This import should be done using ExternalProject_Add. There are numerous examples in the ngraph cmake directory. Add the ONNX importer there.